src/integrationTest/groovy/org/ysb33r/gradle/gradletest/CompileIntegrationSpec.groovy

```diff
@@ -82,7 +82,7 @@ class CompileIntegrationSpec extends Specification {
         generatorTask.didWork
 
         and: "Expected classes in output directory"
-        new File(classesDir,"gradleTest/tests/AlphaGroovyDSLCompatibilitySpec.class").exists()
+        new File(classesDir,"gradleTest/tests/AlphabuildGroovyDSLCompatibilitySpec.class").exists()
```

Comment:
Updating all the tests to the new name pattern for the spec files, this is repeated in all the integration and unit tests.